wire-tests/src/test/proto-kotlin/com/squareup/wire/protos/kotlin/person/Person.kt.java.interop

```diff
@@ -25,7 +25,7 @@ data class Person(
     @field:WireField(tag = 3, adapter = "com.squareup.wire.ProtoAdapter.STRING") val email: String?
             = null,
     @field:WireField(tag = 4, adapter = "PhoneNumber.ADAPTER") val phone: List<PhoneNumber> =
-            emptyList(),
+            Internal.newMutableList(),
```

Comment:
Can you verify if this new output is actually correct?

Reply:
Sure, `./gradlew clean :wire-kotlin-generator:check :wire-compiler:check`